hashmail_server_test.go

```diff
@@ -86,10 +86,10 @@ func TestHashMailServerReturnStream(t *testing.T) {
 
 	// Send then receive yet another message to make sure the stream is
 	// still operational.
-	testMessage2 := append(testMessage, []byte("test")...)
+	testMessage := append(testMessage, []byte("test")...)
```

Comment:
Hmm, this is weird, since it _does_ create a new variable (with the same name but in a new scope). I'd say this might be a candidate for a `//nolint` directive instead, as this looks more like a headscratcher than before IMO.